products/health-care/appointments/va-online-scheduling/research/july-2020-user-research/vaosr-june-research-plan.md

```diff
@@ -1,50 +1,86 @@
+   - Users have information overload/confusion as they are going through the flow to schedule an appointment
+     - Users are looking for, but can't find a specific appointment type, either because it is not listed/supported by online scheduling, at that location, or the user has a different name for the appointment type than what is listed
+     - Users are looking for a specific location, but can't find the specific location they have in mind, either because the location isn't listed or it is listed using a different name/nested within a category they are unfamiliar with
+     - Users are looking for a specific provider, but can't find the specific provider they have in mind
+   - Users are confused by the different look/feel, if this is new to them. Easily give up, rather than trying to figure it out 	
+
 ## Method	
 1.	What method of research are you planning? 	
-  - **Sample Methodologies**: Directed interviews; ethnographic interviews; observation; card sorts; tree tests; guerilla usability testing; usability testing with low- or high-fidelity prototypes; user acceptance testing. 	
-  - **Also mention:** in-person moderated, remote moderated, remote unmoderated	
-  	
-2.	Why this method? How does this methodology help you answer your research questions? 	
+   - Remote moderated usability research and usability testing mid-fidelity prototypes
+2.	Why this method? How does this methodology help you answer your research questions? 
+   -  
 3.	Where are you planning to do your research? *If in person, mention the location, point of contact, arrangements, etc. If online, mention which tool you'll be using (GTM, Join.me, etc.)*	
-4.	What will you be testing? *(Design mocks, card sort, prototype, page, content, etc.)* 	
-5.  If remote: What tool do you plan to use (Zoom, GoToMeeting, Webex)	
+   - Zoom
+4.	What will you be testing? *(Design mocks, card sort, prototype, page, content, etc.)* 
+   - https://www.va.gov/health-care/schedule-view-va-appointments/
+5.	If remote: What tool do you plan to use (Zoom, GoToMeeting, Webex)	
+   - Zoom
+
 ## Participants and Recruitment	
-1.	Participant criteria: What are you looking for in a participant?	
-(Mention: Number of people, ages, accessibility preferences, geographical diversity, login requirements, VA benefit requirements, familiarity with technology, etc. Keep in mind, the more requirements, the more difficult the recruit, so give ample time to ensure the right participant mix.)	
-2.	What is your recruitment strategy? 	
-(If in person, describe how you will find participants. If remote, mention if you plan to draw from the existing recruiting contract or if there are other places where you would like to reach out to find participants specifically for this project. If you need help, please contact UX lead.)	
+1. Participant criteria: What are you looking for in a participant?	
+   - 20	
+   - Ideal, but not required: individuals who need to or who have recently scheduled an appointment
```

Comment:
I think it'd be awesome to have folks who have never scheduled online be a part of this too. they would have an untainted perspective.

Reply:
totally agree! I think this should be a broad recruitment effort, especially since we want such a large number or users. 

I added the optional criteria (maybe rephrasing it to `optional/not required`) after reading the feedback from April-June. Happy to remove it and keep things as broad as possible.

Reply:
removing optional criteria